src/enums/index.ts

```diff
@@ -22,5 +22,5 @@ export * from './affiliationInvitation'
 
 // external enums
 export { CorpTypeCd, CorpClass } from '@bcrs-shared-components/corp-type-module'
-export { AccountTypes, AmalgamationTypes, FilingCodes, FilingNames, FilingTypes, StaffPaymentOptions }
+export { AccountTypes, AmalgamationTypes, FilingCodes, FilingNames, FilingTypes, NameTypes, StaffPaymentOptions }
```

Comment:
Actually, I prefer to see these imported from the shared repo in the components that need them. I'm trying to get past the issue of types not being recognized as the same because they were imported differently. LMK your thoughts on this.

Reply:
Don't spend too much time on this in case this all changes later.

Reply:
I agree Sev. Not only that, but I've noticed A LOT of inconsistency throughout the code. Some files are using these enums from `@/enum` while others are using them from the shared components directly (`@bcrs-shared-components/enums`).

I've made this change. This should also made the code consistent. They're all being used from the shared components now whenever a file needs them.